CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Change Log
 
-## 1.0.2
+## [1.0.3](https://github.com/rangle/augury/tree/1.0.3)
+* Update Augury to Angular 2.0.0-rc.3 ([450](https://github.com/rangle/augury/issues/450))
```

Comment:
Just for my understanding, Augury is built with RC3, and it also supports applications that are written in RC3. @sumitarora